WcaOnRails/config/locales/locales.rb

```diff
@@ -2,18 +2,22 @@
 
 module Locales
   AVAILABLE = {
+    "en": {
```

Comment:
Could you add a comment right above here explaining what the intent of the ordering is? Hopefully future coders will see that and know what to do =)